src/main/java/com/omega/Proyecto/omega/Model/TouristicServ.java

```diff
@@ -0,0 +1,25 @@
+package com.omega.Proyecto.omega.Model;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class TouristicServ {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long idTouristicService;
+    private String name;
+    private String description;
+    private String destiny;
```

Comment:
```suggestion
    private String destination;
```
En inglés "destiny" puede llegar a significar otra cosa diferente al destino de un viaje